tensorflow/lite/experimental/shlo/ops/convolution.h

```diff
@@ -61,9 +61,18 @@ class ConvolutionOp {
     std::array<PrecisionTypes, 2> precision_configs;
   };
   Attributes attributes;
+  Tensor lhs_transposed;
+  Tensor rhs_transposed;
+  Tensor output_transposed;
   Tensor lhs_dot_general;
   Tensor rhs_dot_general;
   Tensor output_dot_general;
+  absl::InlinedVector<Axis, kMaxNumDimensions> lhs_permutations;
+  std::vector<std::byte> lhs_transposed_data;
+  absl::InlinedVector<Axis, kMaxNumDimensions> rhs_permutations;
+  std::vector<std::byte> rhs_transposed_data;
+  absl::InlinedVector<Axis, kMaxNumDimensions> output_permutations;
+  std::vector<std::byte> output_transposed_data;
```

Comment:
change the order as follows:
lhs/rhs/output permutations
and then lhs/rhs/output transposed data